Makefile

```diff
@@ -66,22 +71,17 @@ INTEG_TEST_NDC_ROOT    := ./host/ndc
 PROTO_ROOT := proto
 PROTO_FILES = $(shell find ./$(PROTO_ROOT)/internal -name "*.proto")
 PROTO_DIRS = $(sort $(dir $(PROTO_FILES)))
-PROTO_IMPORTS := -I=$(PROTO_ROOT)/internal -I=$(PROTO_ROOT)/api -I=$(GOPATH)/src/github.com/temporalio/gogo-protobuf/protobuf
+PROTO_IMPORTS := -I=$(PROTO_ROOT)/internal -I=$(PROTO_ROOT)/api -I=$(shell go env GOMODCACHE)/github.com/temporalio/gogo-protobuf@$(GOGO_PROTOBUF_VERSION)/protobuf
```

Comment:
I think a better way to do this is to do
```
go list -f '{{.Dir}}' github.com/temporalio/gogo-protobuf/protobuf
```
to make that work we need an explicit dependency on `github.com/temporalio/gogo-protobuf/protobuf` in go.mod, which should be easy, just added it to PINNED_DEPENDENCIES with that version and add an import in some go file so it's used